src/sdk/signature.ts

```diff
@@ -0,0 +1,20 @@
+export async function verifySignature(publicKeyPem: string, payload: unknown, signature: string) {
+  const pemContents = publicKeyPem.replace("-----BEGIN PUBLIC KEY-----", "").replace("-----END PUBLIC KEY-----", "").trim();
+  const binaryDer = Uint8Array.from(atob(pemContents), (c) => c.charCodeAt(0));
+
+  const publicKey = await crypto.subtle.importKey(
+    "spki",
+    binaryDer,
+    {
+      name: "RSASSA-PKCS1-v1_5",
+      hash: "SHA-256",
+    },
+    true,
+    ["verify"]
+  );
+
+  const signatureArray = Uint8Array.from(atob(signature), (c) => c.charCodeAt(0));
```

Comment:
Isn't `atob` deprecated in favor of `Buffer`, or do you do that for compatibility

Reply:
Aren't these completely different? atob just base64 encodes. Buffer is a binary format or something.

Reply:
From the docs:
> This function is only provided for compatibility with legacy web platform APIs and should never be used in new code, because they use strings to represent binary data and predate the introduction of typed arrays in JavaScript. For code running using Node. js APIs, converting between base64-encoded strings and binary data should be performed using Buffer.from(str, 'base64') and buf.toString('base64').

Reply:
https://developers.cloudflare.com/workers/runtime-apis/nodejs/buffer/
Buffer is only available in NodeJS, so I decided to use atob which is still in cloudflare [docs](https://developers.cloudflare.com/workers/runtime-apis/web-standards/#base64-utility-methods) so I'm assuming it's okay to use